lib/js/components/Outline/OutlineItem/OutlineItem.vue

```diff
@@ -84,6 +91,7 @@
 		align-items: flex-start;
 		column-gap: $space-2xs;
 		display: flex;
+		max-width: 100%;
```

Comment:
what does it change?
When I put a long label without spaces, then text goes outside this component

Reply:
Without it overflow does not work on .katex and .katex-display classes